<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_reportes</class>
 <widget class="QDialog" name="form_reportes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1733</width>
    <height>915</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Estadísticas</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Fecha Inicial</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="de_inicial">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="de_final">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>50</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Fecha Final</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>1261</width>
        <height>381</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox{
	color: black;
}</string>
      </property>
      <property name="title">
       <string>Órdenes</string>
      </property>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>840</x>
         <y>30</y>
         <width>321</width>
         <height>301</height>
        </rect>
       </property>
       <property name="title">
        <string>Totales</string>
       </property>
       <widget class="QLabel" name="l_toralordenes">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>60</y>
          <width>251</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Órdenes:</string>
        </property>
       </widget>
       <widget class="QLabel" name="l_inversion">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>100</y>
          <width>261</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Inversion</string>
        </property>
       </widget>
       <widget class="QLabel" name="l_ingresos">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>140</y>
          <width>251</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Ingresos</string>
        </property>
       </widget>
       <widget class="QLabel" name="l_ganancia">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>190</y>
          <width>251</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Ganancia</string>
        </property>
       </widget>
      </widget>
      <widget class="QTableWidget" name="tw_ordenes">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>761</width>
         <height>321</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>Fecha</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>No. Orden</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Pagado</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Inversion</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ingresos</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ganancia</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>480</y>
        <width>1261</width>
        <height>391</height>
       </rect>
      </property>
      <property name="title">
       <string>Gastos</string>
      </property>
      <widget class="QTableWidget" name="tw_gastos">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>751</width>
         <height>331</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>Fecha</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Descripción</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Importe</string>
        </property>
       </column>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>840</x>
         <y>30</y>
         <width>321</width>
         <height>181</height>
        </rect>
       </property>
       <property name="title">
        <string>Totales</string>
       </property>
       <widget class="QLabel" name="l_totaleventos">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>241</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Eventos</string>
        </property>
       </widget>
       <widget class="QLabel" name="l_importeeventos">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>70</y>
          <width>241</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Importe Total</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>1370</x>
        <y>90</y>
        <width>321</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Totales Generales</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QLabel" name="l_onat">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>241</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ONAT</string>
       </property>
      </widget>
      <widget class="QLabel" name="l_ganancianeta">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>271</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Ganancia Neta</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pb_filtrar">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>50</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Filtrar</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
